spec/index.bs

```diff
@@ -917,7 +917,8 @@ interface GPUTextureUsage {
     const GPUTextureUsageFlags COPY_DST          = 0x02;
     const GPUTextureUsageFlags SAMPLED           = 0x04;
     const GPUTextureUsageFlags STORAGE           = 0x08;
-    const GPUTextureUsageFlags OUTPUT_ATTACHMENT = 0x10;
+    const GPUTextureUsageFlags READONLY_STORAGE  = 0x10;
```

Comment:
I don't think this usage is needed: see #541

Reply:
wait, you don't see that this is needed?
confused because you are pointing to the issue saying that it is needed :)

Reply:
Following discussions in the meeting, can we land this PR and discuss readonly usage flags for buffers and textures in a separate PR?

Reply:
Editors concluded to slightly prefer `STORAGE_READ` name, other than that we can land

Reply:
Done

Reply:
If we have `STORAGE_READ` should we not have `STORAGE = 0x18`?

If we decide there's no chance of adding a `STORAGE_WRITE` later (TBD), I'd prefer `READONLY_STORAGE`.

Reply:
AH right, thanks for the clarification. My take was that `STORAGE_READ` is just more consistent with other usage names and future-proof for the case where we want to have `STORAGE_WRITE`. I didn't realize you have a strong preference to `READONLY_STORAGE`, this would mean we have to resolve the write-only conversation first before proceeding with this one...

Reply:
To clarify, mainly would prefer `READONLY_STORAGE` if we end up with `"readonly-storage-binding"`, for consistency.